README.md

```diff
@@ -5,7 +5,7 @@ For better ergonomics with xtra, try the [spaad](https://crates.io/crates/spaad)
 
 ## Features
 - Safe: there is no unsafe code in xtra.
-- Tiny: xtra is less than 1k loc.
+- Tiny: xtra is less than 2kloc.
```

Comment:
This can be updated again ahead of 0.6 :) Hopefully some convenience methods can be moved out to `xtras` or similar, in order to make the core library smaller, lighter, and more maintainable!